src/openforms/emails/confirmation_emails.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Do we need that in python 3.12? Or are we doing this for earlier versions?

Reply:
this mostly makes the annotations/imports lazy so you can use the `Name` in places when doing conditional imports without having to wrap them in strings: `"Name"`